proposals/lip-Introduce_cross-chain_update_mechanism.md

```diff
@@ -0,0 +1,620 @@
+                "messageWitness", 
+                "outboxRootWitness"
+            ],
+            "properties": {
+                "crossChainMessages": {
+                    "type": "array",
+                    "fieldNumber": 1,
+                    "items": {"dataType": "bytes"}
+                },
+                "messageWitness": {
+                    "type": "object",
+                    "fieldNumber": 2,
+		    "required": ["partnerChainOutboxSize", "siblingHashes"],
+                    "properties": { 
+                        "partnerChainOutboxSize":{
+                            "dataType": "uint64",
+                            "fieldNumber": 1
+                        },
+                        "siblingHashes":{
+                            "type": "array",
+                            "fieldNumber": 2,
+                            "items": {"dataType": "bytes"}
+                        }
+                    }
+                },
+                "outboxRootWitness": {
+                    "type": "object",
+                    "fieldNumber": 3,
+                    "required": ["bitmap", "siblingHashes"],
+                    "properties": {
+                        "bitmap":{
+                            "dataType": "bytes",
+                            "fieldNumber": 1
+                        },
+                        "siblingHashes":{
+                            "type": "array",
+                            "fieldNumber": 2,
+                            "items": {"dataType": "bytes"}
+			}
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+
+### Parameters Validity
+
+In the following, let `sendingAccount` be the entry in the interoperability store with store prefix `STORE_PREFIX_CHAIN_DATA` and store key `uint32be(sendingChainID)`.
+
+
+#### Liveness of Sending Chain
+
+A CCU transaction is only valid if the sending chain is not terminated and follows the [liveness rule][base-interoperability-LIP-livenessCondition]. 
+This is done by asserting the two points below:  
+
+*  `sendingChain.status != CHAIN_TERMINATED`,
+*  if `sendingChain.status == CHAIN_ACTIVE` then also validate  `isLive(sendingChainID,timestamp) == True`. 
+   The `isLive` function is specified in [LIP "Introduce Interoperability module"][base-interoperability-LIP], and `timestamp` is the timestamp of the block including the CCU.
+
+
+##### Liveness Requirement for the First CCU
+
+If `sendingChain.status == CHAIN_REGISTERED`, the proposed CCU must contain a non-empty `certificate` which must follow the schema defined in  [LIP "Introduce a certificate generation mechanism"][certificate-generation-LIP]. 
+In the following, let `certificate` be the deserialized certificate.
+
+Furthermore, the certificate is only valid if it allows the sidechain account to remain live for a reasonable amount of time.
+This is done by checking that
+```java
+timestamp - CCU.params.certificate.timestamp < 15*24*3600
```

Comment:
In the interoperability LIP I introduced the constant `LIVENESS_LIMIT` for this. We could add it here to the table as well

Reply:
Also maybe use python formatting

Reply:
changed formatting.
changed to use `LIVENESS_LIMIT`.